pyhms/utils/print_tree.py

```diff
@@ -0,0 +1,34 @@
+import numpy as np
+
+from ..demes.abstract_deme import AbstractDeme
+
+LEVEL_PREFIX = "-- "
+LAST_LEVEL_SYMBOL = "\u2514"
+MIDDLE_LEVEL_SYMBOL = "\u251C"
+
+
+def format_array(solution: np.ndarray, float_format="{:#.2f}") -> str:
+    solution_values = [float_format.format(sol) for sol in solution]
+    return "(" + ", ".join(solution_values) + ")"
+
+
+def format_deme(deme: AbstractDeme, best_fitness: float | None = None) -> str:
+    deme_distinguisher = "***" if best_fitness and deme.best_individual.fitness == best_fitness else ""
```

Comment:
When applying the method to the 3-level trees (I modified test_more_levels.py), I don't get the distinguisher for best deme:
```
f(-1.62, 1.91) = 6.27e+00  evals: 220
├-- f(-0.10, 0.02) = 1.09e-02 spr: (-1.62, 1.91); evals: 200
|   ├-- f(-0.00, -0.00) = 1.28e-06 spr: (-0.10, 0.02); evals: 174
|   ├-- f(-0.00, -0.00) = 7.89e-06 spr: (-0.10, 0.02); evals: 150
|   └-- f(0.00, -0.00) = 1.97e-06 spr: (-0.10, 0.02); evals: 126
├-- f(-0.01, -0.04) = 1.51e-03 spr: (-1.62, 1.91); evals: 180
|   └-- f(-0.00, 0.00) = 5.14e-06 spr: (-0.37, 0.14); evals: 150
├-- f(-0.04, -0.26) = 6.81e-02 spr: (-1.62, 1.91); evals: 160
└-- f(0.18, -0.03) = 3.36e-02 spr: (-1.62, 1.91); evals: 140
```
Btw. as you can see demes are sprouted at the same seed. Although they may just move out of their starting location after one metaepoch, this may also be a critical issue. I will look into it today and rewrite sprout_tests

Reply:
Right, thanks! I forgot to pass `best_fitness` to `print_tree_from_deme`.